### PR TITLE
ISPN-7135

### PR DESCRIPTION
**Pull request by wburns**
ClusterListenerReplTest.testPrimaryOwnerGoesDownBeforeBackupRaisesEvent
random failures

* Change test to allow for 2 retries

https://issues.jboss.org/browse/ISPN-7135

### CODE REVIEW

File: core/src/test/java/org/infinispan/notifications/cachelistener/cluster/ClusterListenerReplTest.java [hunk 1]
```diff
@@ -122,10 +114,13 @@ public void testPrimaryOwnerGoesDownAfterBackupRaisesEvent() throws InterruptedE
       String returnValue = future.get(10, TimeUnit.SECONDS);
       assertEquals(FIRST_VALUE, returnValue);
 
-      assertEquals(2, clusterListener.events.size());
+      assertTrue(clusterListener.events.size() >= 2);
+      sassertTrue(clusterListener.events.size() <= 3);
```
Comment (pruivo) [inline core/src/test/java/org/infinispan/notifications/cachelistener/cluster/ClusterListenerReplTest.java]: I think ```sassert```doesn't exist :)

Comment (wburns) [inline core/src/test/java/org/infinispan/notifications/cachelistener/cluster/ClusterListenerReplTest.java]: thanks 👍 Intellij does when I press ctrl-s but ctrl wasn't held down long enough :)

**Comment by wburns** [comment]
Fixed

**Comment by pruivo** [comment]
integrated! thanks @wburns

**Pull request merged**